packages/where/index.js

```diff
@@ -59,32 +56,32 @@ export const whereEndoSock = (platform, env) => {
     } else {
       return raw`\\?\pipe\Endo\endo.pipe`;
     }
+  } else if (env.XDG_RUNTIME_DIR !== undefined) {
+    return `${env.XDG_RUNTIME_DIR}/endo/endo.sock`;
   } else if (platform === 'darwin') {
     if (env.HOME !== undefined) {
       return `${env.HOME}/Library/Application Support/Endo/endo.sock`;
     }
-  } else if (env.XDG_RUNTIME_DIR !== undefined) {
-    return `${env.XDG_RUNTIME_DIR}/endo/endo.sock`;
   } else if (env.USER !== undefined) {
     return `/tmp/endo-${env.USER}/endo.sock`;
   }
-  return 'endo.sock';
+  return 'endo/endo.sock';
 };
 
 /**
  * @type {typeof import('./types.js').whereEndoCache}
  */
 export const whereEndoCache = (platform, env) => {
```

Comment:
yay for explicit `platform, env` args rather than ambient access to `os.type()` and `process.env`.

Reply:
Yay, testable!